recipes/r-canopy/meta.yaml

```diff
@@ -46,15 +43,7 @@ requirements:
 
 test:
   commands:
-    # You can put additional test commands to be run here.
-    - $R -e "library('Canopy')"  # [not win]
```

Comment:
This line has to stay - it's used for testing on non-windows systems.

Reply:
Sorry if I wasn't clear in my last comments on the `# [win]` / `# [not win]` part..